### ISSUE TITLE
REPLACE_ENV_VARIABLES takes very long

### ISSUE THREAD

**Issue by Trikolon** [itzg/docker-minecraft-server#575]
I have quite a big server with a lot of playerdata saved in `.yml` files. That causes the server start to take over 15 minutes because the container has to go through all the files to search-replace.
Would it be possible to add an include / exclude list for this replace operation?

**Comment by itzg** [comment]
I just wanted to let you know that I have started to think through how to fix the issues you reported. I'm hoping to have these fixed within the next few days.

**Comment by itzg** [comment]
The latest image now provides an option for space-delimited listing filenames to exclude from processing: `REPLACE_ENV_VARIABLES_EXCLUDES`:

https://github.com/itzg/docker-minecraft-server#replacing-variables-inside-configs

**Comment by Trikolon** [comment]
Thanks for providing such a quick fix! However, I think this doesn't quite solve my issue.

A typical case is a plugin which stores configuration per-player and creates one  `.yml` file for every player, the filename being their UUID. For example: `069a79f4-44e9-4726-a5be-fca90e38aaf5.yml`. I can't really explictly list these.

Would it be possible to add path excludes? For example:
```
REPLACE_ENV_VARIABLES_EXCLUDE_PATHS="plugins/Essentials/userdata/"
```

**Comment by itzg** [comment]
Gotcha. Yes, I should be able to craft the `find` option to do a path based exclusion. Stay tuned...

**Comment by itzg** [comment]
I have now added `REPLACE_ENV_VARIABLES_EXCLUDE_PATHS`, so your example snippet should work now.

**Comment by Trikolon** [comment]
Sorry for the delay. I've just tested the exclude paths option.

You're wrapping the paths in `*` here: https://github.com/itzg/docker-minecraft-server/blob/3b2b98b9fe72d46e5a264a139f5de254baca1ee8/start-finalSetupEnvVariables#L15
This breaks cases where I can't use wildcards because the path isn't very unique. For example if my world is called `survival` and is in the base directory. Perhaps it would be better to leave that up to the user and explain how the exclude works in the documentation.

The second issue is that the search replace is still really slow. I suspect this might be because `find` still dives into all these excluded directories. Maybe the prune option would fix this (we've talked about it on Discord).
